update

FedorovMykhailo opened the pull request:
Доданий виклик редагування запису випитої води, стилі для планшету та мобільної версії модального вікна випитої води, дописана процедура редагування запису випитої води, додані хтмл атрибути інпутів для перевірки введених значень.

FedorovMykhailo committed:
update

Доданий виклик редагування запису випитої води, стилі для планшету та мобільної версії модального вікна випитої води,
дописана процедура редагування запису випитої води, додані хтмл атрибути інпутів для перевірки введених значень.

Supreme-cake-being reviewed (changes requested):
Загубилися відступи від кнопок
Відкривається модалка Edit при додавані 
Прибери am pm у TodayWaterList та просто залиш час як він з беку приходить або зроби перевірку на те щоб якщо ми вводимо 00:00, то нам не виводилося 12:00 am

FedorovMykhailo committed:
update TodayListModal

Поправив косяки, тепер візуал модалок різниться за наповненням. Додав сортування стейту за часом для "більш логічного" відображення записів.  трек інпут рейндж тепер "підсвічує" прогрес пиття:) Початкові значення в модалках тепер згідно умов ТЗ.

Supreme-cake-being committed:
added margins for TodayWaterList component

FedorovMykhailo committed:
Merge branch 'main' into origin/WaterRatioPanel
